code/datums/underwear/undershirt.dm

```diff
@@ -0,0 +1,447 @@
+	name = "Shortsleeve Undershirt"
+	icon_state = "shortsleeve"
+	has_color = TRUE
+
+/datum/category_item/underwear/undershirt/shortsleeve_undershirt_fem
+	name = "Shortsleeve Undershirt, Feminine"
+	icon_state = "shortsleeve_fem"
+	has_color = TRUE
+
+/datum/category_item/underwear/undershirt/shortsleeve_undershirt_vneck
+	name = "Shortsleeve Undershirt, V-Neck"
+	icon_state = "shortsleeve_vneck"
+	has_color = TRUE
+
+/datum/category_item/underwear/undershirt/shortsleeve_undershirt_vneck_fem
+	name = "Shortsleeve Undershirt, V-Neck Feminine"
+	icon_state = "shortsleeve_vneck_fem"
+	has_color = TRUE
+
+/datum/category_item/underwear/undershirt/undershirt
+	name = "Undershirt"
+	icon_state = "undershirt"
+	has_color = TRUE
+
+/datum/category_item/underwear/undershirt/undershirt_fem
+	name = "Undershirt, Feminine"
+	icon_state = "undershirt_fem"
+	has_color = TRUE
+
+/datum/category_item/underwear/undershirt/undershirt_heart
+	name = "Undershirt, Heart"
+	icon_state = "heart"
+
+/datum/category_item/underwear/undershirt/undershirt_heart_fem
+	name = "Undershirt, Heart Feminine"
+	icon_state = "heart_fem"
+
+/datum/category_item/underwear/undershirt/undershirt_heart_echelon
+	name = "Undershirt, Heart Echelon"
+	icon_state = "heart_echelon"
+
+/datum/category_item/underwear/undershirt/undershirt_heart_echelon_fem
+	name = "Undershirt, Heart Echelon Feminine"
+	icon_state = "heart_echelon_fem"
+
+/datum/category_item/underwear/undershirt/shirt_love_nt
+	name = "Undershirt, I<3NT"
+	icon_state = "ilovent"
+
+/datum/category_item/underwear/undershirt/shirt_love_nt_fem
+	name = "Undershirt, I<3NT Feminine"
+	icon_state = "ilovent_fem"
+
+/datum/category_item/underwear/undershirt/shirt
+	name = "Shirt"
+	icon_state = "shirt"
+	has_color = TRUE
+
+/datum/category_item/underwear/undershirt/shirt_alien
+	name = "Shirt, Alien"
+	icon_state = "shirt_alien"
+
+/datum/category_item/underwear/undershirt/shirt_band
+	name = "Shirt, Band"
+	icon_state = "shirt_band"
+
+/datum/category_item/underwear/undershirt/shirt_clown
+	name = "Shirt, Clown"
+	icon_state = "shirt_clown"
+
+/datum/category_item/underwear/undershirt/shirt_ian
+	name = "Shirt, Ian"
+	icon_state = "shirt_ian"
+
+/datum/category_item/underwear/undershirt/shirt_jersey
+	name = "Shirt, Blue Jersey"
+	icon_state = "shirt_jersey"
+
+/datum/category_item/underwear/undershirt/shirt_jersey_red
+	name = "Shirt, Red Jersey"
+	icon_state = "shirt_jersey_red"
+
+/datum/category_item/underwear/undershirt/shirt_meat
+	name = "Shirt, Meat"
+	icon_state = "shirt_meat"
+
+/datum/category_item/underwear/undershirt/shirt_nt
+	name = "Shirt, NT"
+	icon_state = "shirt_nano"
+
+/datum/category_item/underwear/undershirt/shirt_nt_red
+	name = "Shirt, NT Red"
+	icon_state = "shirt_nano_red"
+
+/datum/category_item/underwear/undershirt/shirt_peace
+	name = "Shirt, Peace"
+	icon_state = "shirt_peace"
+
+/datum/category_item/underwear/undershirt/shirt_pogoman
+	name = "Shirt, Pogoman"
+	icon_state = "shirt_pogoman"
+

```

Comment:
Sprite is missing, either delete the datum or add the sprite.

Reply:
I could've sworn I removed the sprites too...